ibm/resource_ibm_is_lb.go

```diff
@@ -985,7 +985,7 @@ func isWaitForLBAvailable(sess *vpcv1.VpcV1, lbId string, timeout time.Duration)
 	log.Printf("Waiting for load balancer (%s) to be available.", lbId)
 
 	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"retry", isLBProvisioning},
+		Pending:    []string{"retry", isLBProvisioning, "update_pending", "delete_pending"},
```

Comment:
Update_pending states are not needed , and the refresh function returns "provisioning" for all other states except for 'active' and 'failed'

Reply:
Hi @deepaksibm I am using the load_balancer's resource function. So I can't remove anything from there because it is used for load_balancer creation as well.

Reply:
why delete_pending is addeed for active status

Reply:
@hkantare Added just like that..Should I remove it??